synapseAnnotations/data/sageCommunity.json

```diff
@@ -454,6 +474,36 @@
         "value": "RData",
         "description": "The RData format (usually with extension .rdata or .rda) is a format designed for use with R, a system for statistical computation and related graphics, for storing a complete R workspace or selected 'objects' from a workspace in a form that can be loaded back by R.",
         "source": "https://www.loc.gov/preservation/digital/formats/fdd/fdd000470.shtml"
+      },
+      {
+        "value": "ab1",
+        "description": "TAB1 binary format of raw DNA sequence reads (output of Applied Biosystems' sequencing analysis software). Contains an electropherogram and the DNA base sequence.",
+        "source": "http://edamontology.org/format_3000"
+      },
+      {
+        "value": "7z",
+        "description": "7z is a compressed archive file format that supports several different data compression, encryption and pre-processing algorithms.",
+        "source": "https://en.wikipedia.org/wiki/7z"
```

Comment:
Maybe http://purl.obolibrary.org/obo/NCIT_C80224 instead?

Reply:
Yeah...I saw this one but "7Z Compression Format" seems so excessive!

I'll make the change, though

Reply:
It's ok with me to keep the value as "7Z" even though the preferred term is "7Z Compression Format". This is what we did with "gzip", since we're already inconsistent on this with other fileFormat annotations (like tsv) and many of our fileFormat values don't come from ontologies at all.